SPECS/php/php-8.1.11.tar.xz.asc

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PGP SIGNATURE-----
```

Comment:
Please remove this file along with `php-keyring.gpg`. I think they are not used in the spec at all and we rely on the signatures anyway. When we do have a PGP signature, we can verify it once, save the hash of the source with other signatures and that should be all.

Reply:
Not sure why the source macros disappeared, but the PHP spec should have
```
# See https://secure.php.net/gpg-keys.php
Source20: https://www.php.net/distributions/php-keyring.gpg
Source21: https://www.php.net/distributions/php-%{version}.tar.xz.asc
```
as well as
```
%prep
%{?gpgverify:%{gpgverify} --keyring='%{SOURCE20}' --signature='%{SOURCE21}' --data='%{SOURCE0}'}
```
Should I just remove these parts of the code entirely then?

Reply:
Yep!